### ISSUE TITLE
Undeclared dependency on pandas

### ISSUE THREAD

**Issue by mr-martian** [annotation/text-fabric#81]
After running `pip3 install -e .` on the most recent commit, I got the following traceback:

```
Traceback (most recent call last):
  File "/home/daniel/./tf_test.py", line 3, in <module>
    from tf.app import use
  File "/home/daniel/text-fabric/tf/app.py", line 16, in <module>
    from .advanced.app import findApp, useApp
  File "/home/daniel/text-fabric/tf/advanced/app.py", line 29, in <module>
    from .interchange import interchangeApi
  File "/home/daniel/text-fabric/tf/advanced/interchange.py", line 20, in <module>
    from ..convert.pandas import exportPandas
  File "/home/daniel/text-fabric/tf/convert/pandas.py", line 36, in <module>
    import pandas
ModuleNotFoundError: No module named 'pandas'
```

It would appear that something in the code requires pandas to be installed, but this is not listed in the packaging dependencies.

**Comment by dirkroorda** [comment]
Will do that or either I will indicate in the docs that pandas must be installed if you want to do big tables.

**Comment by dirkroorda** [comment]
Ah, I had `pandas` as an install option in TF:

```
pip install text-fabric[pandas]
```

I want to be able to install text-fabric without the dependency on Pandas.
Most users do not need this, and I think iPad users cannot install TF if it is dependent on Pandas.
So if you need pandas, you have to install it yourself.

That was the reasoning.
But now I see that when importing tf.app you need to import pandas.
Ok, I'll add it to the dependencies.

**Comment by dirkroorda** [comment]
Fixed in 12.1.3
